libs/feature/record/src/lib/data-view/data-view.component.spec.ts

```diff
@@ -146,4 +159,55 @@ describe('DataViewTableComponent', () => {
       })
     })
   })
+  describe('initial state in chart mode', () => {
```

Comment:
I don't think it's necessary to duplicate the whole test here, this seems like a lot of duplicate code; you could probably just check that the component correctly creates a chart-view or table-view depending on its type